addons/mail/static/src/models/composer/composer.js

```diff
@@ -1031,10 +1165,12 @@ function factory(dependencies) {
         isLog: attr({
             default: false,
         }),
+        isProcessingQueue: attr({
```

Comment:
`isSendingMessages` + docstring
Also if the queue is on thread, this field should probably be on thread too.